bedrock/firefox/views.py

```diff
@@ -68,7 +69,14 @@ def installer_help(request):
     if installer_lang and installer_lang in firefox_desktop.languages:
         context['installer_lang'] = installer_lang
 
+    if installer_channel and installer_channel == 'aurora':
+        context['installer_channel'] = 'alpha'
+
     if installer_channel and installer_channel in INSTALLER_CHANNElS:
+        if installer_channel == 'aurora':
+            context['installer_channel'] = 'alpha'
+        else:
+            context['installer_channel'] = installer_channel
         context['installer_channel'] = installer_channel
```

Comment:
This is overriding the conditional you've added above. I think if you remove this line then things should work ok.